src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
 function globalInitialization(context, callback) {
     process.on('uncaughtException', function (err) {
         context.handleUncaughtException(err.stack);
+        // delay exit to allow .net unhandled exception logging to complete
+        setTimeout(() => process.exit(1), 500);
```

Comment:
I don't see why this line is needed or how it helps? The above call logs the error and flushes the logs. It also rethrows the exception killing the process.